chat/src/main/java/net/md_5/bungee/chat/SelectorComponentSerializer.java

```diff
@@ -22,6 +22,12 @@ public SelectorComponent deserialize(JsonElement element, Type type, JsonDeseria
             throw new JsonParseException( "Could not parse JSON: missing 'selector' property" );
         }
         SelectorComponent component = new SelectorComponent( object.get( "selector" ).getAsString() );
+
+        if ( object.has( "separator" ) )
+        {
+            component.setSeparator( ComponentSerializer.parse( object.get( "separator" ).getAsString() ) );
```

Comment:
Consequently, this should call `ComponentSerializer#deserialize()` instead of `#parse()`